src/c/admin-list.js

```diff
@@ -29,7 +29,7 @@ window.c.AdminList = (function(m, h, c){
               ]),
               m('#admin-contributions-list.w-container',[
                 list.collection().map(function(item) {
-                  return m.component(c.AdminItem, {builder: args.itemBuilder, actions: args.itemActions, item: item, key: item.key});
+                  return m.component(c.AdminItem, {builder: args.itemBuilder, actions: args.itemActions, item: m.prop(item), key: item.key});
```

Comment:
Ok, that's easy to setup in our API :D
I thought of this propified item to propagate the change from the component that calls the patch.